sky/clouds/lambda_cloud.py

```diff
@@ -0,0 +1,245 @@
+"""Lambda Cloud."""
```

Comment:
I would prefer to keep this file to be `lambda.py`, because we don't call `aws_cloud`.

Reply:
The downside here is that `from sky.clouds.lambda import Lambda` is invalid syntax :(

Reply:
Ahh, that makes sense! Thanks!